### PR TITLE
DOC-572 fix with SSL improvements and compatibility matrix

### PR DESCRIPTION
**Pull request by fiowro**

### CODE REVIEW

File: doc/source/management/ssl_compatibility.rst [hunk 1]
```diff
@@ -0,0 +1,60 @@
+SSL improvements
+================
+
+|PS| have undergone several SSL compatibility improvements, distinguishing it
+from the original MySQL commuinty edition.
```
Comment (laurynas-biveinis) [inline doc/source/management/ssl_compatibility.rst]: commuinty

File: doc/source/management/ssl_compatibility.rst [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+|PS| have undergone several SSL compatibility improvements, distinguishing it
+from the original MySQL commuinty edition. 
+
+Linking with OpenSSL
+--------------------
+
+As stated in `<https://dev.mysql.com/doc/refman/5.5/en/openssl-versus-yassl.html>`_,
+MySQL can be compiled using either OpenSSL toolkit or yaSSL embedded library to
+enable encrypted connections based on the OpenSSL API. All binary packages of
```
Comment (laurynas-biveinis) [inline doc/source/management/ssl_compatibility.rst]: should be discussing network protocols, not APIs: encrypted connections based on Transport Layer Security / Secure Sockets Layer

File: doc/source/management/ssl_compatibility.rst [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+disabled by default. The reason is The PCI Security Standards compliance, as
+PCI Security Standards Council made TLS 1.0 deprecated starting from May 2016,
+and have stated 30 June 2018 as a deadline for disabling TLS 1.0 in favor of a
+more secure encryption protocol, i.e. TLS 1.1 or higher (TLS v1.2 is strongly
+encouraged).
+
+Subject Alternative Names and binlog SSL options support
+--------------------------------------------------------
+
+Due to using OpenSSL version not older than 1.1, *Percona Server for MySQL*
```
Comment (laurynas-biveinis) [inline doc/source/management/ssl_compatibility.rst]: Drop the "Due ... " clause

Comment (laurynas-biveinis) [inline doc/source/management/ssl_compatibility.rst]: OpenSSL 1.1 support is an item by itself

File: doc/source/management/ssl_compatibility.rst [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+-------------------------------------
+
+*Percona Server for MySQL* supports TLS version 1.2, and the version 1.0 is
+disabled by default. The reason is The PCI Security Standards compliance, as
+PCI Security Standards Council made TLS 1.0 deprecated starting from May 2016,
+and have stated 30 June 2018 as a deadline for disabling TLS 1.0 in favor of a
+more secure encryption protocol, i.e. TLS 1.1 or higher (TLS v1.2 is strongly
+encouraged).
+
+Subject Alternative Names and binlog SSL options support
```
Comment (laurynas-biveinis) [inline doc/source/management/ssl_compatibility.rst]: binlog SSL options support not in this section

File: doc/source/management/ssl_compatibility.rst [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+| SAN validation          | No    | Yes             | Yes              |
++-------------------------+-------+-----------------+------------------+
+| Wildcards support       | No    | No              | Yes              |
++-------------------------+-------+-----------------+------------------+
+
+SSL support in ``mysqlbinlog``
+------------------------------
+
+Also ``mysqlbinlog`` utility in Percona Server includes support for client-side
+SSL options. All encryption-aware client tools beside mysqlbinlog in MySQL are
```
Comment (laurynas-biveinis) [inline doc/source/management/ssl_compatibility.rst]: What are they?

**Review by laurynas-biveinis — CHANGES_REQUESTED** [review]

**Comment by fiowro** [comment]
Fixed all current comments :)

File: doc/source/management/ssl_compatibility.rst [hunk 6]
```diff
@@ -0,0 +1,63 @@
...
+| SAN validation          | No    | Yes             | Yes              |
++-------------------------+-------+-----------------+------------------+
+| Wildcards support       | No    | No              | Yes              |
++-------------------------+-------+-----------------+------------------+
+
+SSL support in ``mysqlbinlog``
+------------------------------
+
+Also ``mysqlbinlog`` utility in Percona Server includes support for client-side
+SSL options. Other encryption-aware client tools (``mysql``, ``mysqlcheck``,
```
Comment (laurynas-biveinis) [inline doc/source/management/ssl_compatibility.rst:58]: No, the comment was not to list the other command line tools, but the added SSL options

File: doc/source/management/ssl_compatibility.rst [hunk 7]
```diff
@@ -0,0 +1,63 @@
...
+enable encrypted connections based on Transport Layer Security / Secure Sockets
+Layer protocols. All binary packages of *Percona Server for MySQL* are linked
+with system OpenSSL, whereas MySQL community edition binary is linked with
+yaSSL. 
+
+Therefore system administrator should take into account that *Percona Server
+for MySQL* is not dependent on yaSSL security advisories, but instead depends
+on system OpenSSL ones, which are to be tracked to make promptly upgrade as
+necessary. Functionality differences between these two SSL backends are covered
+in `<https://dev.mysql.com/doc/refman/5.6/en/openssl-versus-yassl.html>`_.
```
Comment (laurynas-biveinis) [inline doc/source/management/ssl_compatibility.rst:21]: 5.5 link please

**Review by laurynas-biveinis — CHANGES_REQUESTED** [review]
OpenSSL 1.1 support nowhere mentioned as a separate item

**Comment by patrickbirch** [comment]
Closing old PR for 5.5.

**Pull request closed without merging**